cmdeploy/src/cmdeploy/tests/test_cmdeploy.py

```diff
@@ -25,11 +25,17 @@ def test_parser(self, capsys):
 
     def test_init_not_overwrite(self, capsys):
```

Comment:
i believe you could use the "capfd" fixture which captures on the file-descriptor level instead of on the python level of 'sys.std*' that "capsys" provides. 

This should also resolve #622 without needing to do "is_pytest()" and similar shenenigans.